### PR TITLE
Add CUnit 2.1-3

### PR DESCRIPTION
**Pull request by alarcher**
Required for unit testing of upcoming OpenBLAS.

### CODE REVIEW

**Comment by 0xffea** [comment]
LGTM

File: components/cunit/cunit.p5m [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/doc/CUnit/CUnit_doc.css
```
Comment (pyhalov) [inline components/cunit/cunit.p5m]: It's wrong to deliver files in /usr/doc. Documentation should live in /usr/share/doc

**Comment by alarcher** [comment]
OK. I missed it. I will fix that.

**Comment by alarcher** [comment]
Added patch 04 to fix the docdir path.
Enabled test which consists in running test_cunit.
In the manifest, mode=0555 set to test binaries shipped with the package.
Thank you.

**Pull request merged**